integration-tests/js-compute/fixtures/app/src/dynamic-backend.js

```diff
@@ -1527,7 +1522,7 @@ routes.set("/backend/timeout", async () => {
     routes.set("/backend/health/called-as-constructor-function", async () => {
       let error = assertThrows(() => {
         new Backend.health()
-      }, TypeError, `Backend.health is not a constructor`)
+      }, TypeError)
```

Comment:
This is now `(intermediate value).health is not a constructor` (but still pointing to the line number source of the error in the original source, so the stack retains the context). This is due to the ad-hoc nature of the bytecode name tracing in https://searchfox.org/mozilla-central/source/js/src/vm/BytecodeUtil.cpp#1590, which doesn't carry through to the expression naming for the builtin module. The same thing happens for arbitrary JS expressions though anyway (eg `new getObj().health()`).

Since `new Backend.health()` is a pretty extreme edge case, as most JS developers won't attempt to call a function as a class constructor, so I don't think this error message change would be considered breaking.